src/client.js

```diff
@@ -1,8 +1,8 @@
-import Discord from 'discord.js';
 import { dispatchCommand } from './dispatcher';
 import { get } from './environment';
 import logger from 'winston';
 import mongoose from 'mongoose';
+import Discord, { DMChannel } from 'discord.js';
```

Comment:
feels like a weird syntax but is valid I guess

Reply:
this is actually very common in javascript. you'll see it almost all the time, especially when working with react or vue... or anything in es6 haha